### ISSUE TITLE
[QUESTION] Loss function

### ISSUE THREAD

**Issue by sravan953** [yqx7150/RicianNet#1]
Hello,

What is the loss function used to train the model?
